front/src/pods/master/components/show-voting-results.component.tsx

```diff
@@ -12,13 +13,14 @@ export const ShowVotingResults: React.FC<Props> = props => {
   return (
     <>
       <span>Show Voting results</span>
-      <ul>
+      <TablePlayerComponent playersCollection={voteCollectionResult} />
+      {/* <ul>
         {voteCollectionResult.map(({ nickname, vote }) => (
           <li key={nickname}>
             {nickname} - {vote}
           </li>
         ))}
-      </ul>
+      </ul> */}
```

Comment:
I think you let here a lechuguita (commented code), I think this one should be removed?

Reply:
Is removed, I resolve in next pull